src/Sarif.Converters/PREFastConverter.cs

```diff
@@ -57,22 +43,15 @@ public override void Convert(Stream input, IResultLogWriter output, OptionallyEm
                     results.Add(CreateResult(entry));
                 }
 
-                var fileInfoFactory = new FileInfoFactory(MimeType.DetermineFromFileExtension, dataToInsert);
-                Dictionary<string, FileData> fileDictionary = fileInfoFactory.Create(results);
-
-                if (fileDictionary?.Any() == true)
+                var run = new Run()
                 {
-                    output.WriteFiles(fileDictionary);
-                }
+                    Tool = new Tool { Name = ToolFormat.PREfast, FullName = "PREfast Code Analysis" },
+                    ColumnKind = ColumnKind.Utf16CodeUnits
```

Comment:
>ColumnKind = ColumnKind.Utf16CodeUnits [](start = 20, length = 38)

You could DRY this out by adding
```
static Run Create()
{
    return new Run { ColumnKind = ColumnKind.Utf16CodeUnits; }
}
```
... to the Run.cs partial class in Sarif\Core.

Reply:
... It's true that then you couldn't use property initialization syntax to fill the rest of the properties, so... whatever you like.

---
In reply to: [244521253](https://github.com/Microsoft/sarif-sdk/pull/1191#discussion_r244521253) [](ancestors = 244521253)

Reply:
We should make a pass over converter run creation in the future to ensure we fully hydrate things like full name.


---
In reply to: [244521261](https://github.com/Microsoft/sarif-sdk/pull/1191#discussion_r244521261) [](ancestors = 244521261,244521253)